inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -141,6 +142,18 @@ def start(self, tls: bool = True) -> None:
         with ota_lock:
             self._perform_startup_tasks()
 
+        # Run scheduler to schedule the task
+        scheduler = APScheduler()
+        sqliteManager = SqliteManager()
+        single_schedule_list = sqliteManager.get_single_schedule_manifest()
```

Comment:
this could be done with a JOIN -- you would want to select from single_schedule_manifest but then join single_schedule on the schedule_id key, and also join manifest on the manifest_id key

Reply:
Implementation updated.